src/main/java/com/twilio/type/InboundSmsPrice.java

```diff
@@ -22,7 +22,7 @@
     public enum Type {
         LOCAL("local"),
         MOBILE("mobile"),
-        NATIONAL("national"),
+        SHORTCODE("shortcode"),
```

Comment:
Removing the enum type could be breaking for customers even if the API no longer returns it. Think the new type should be added instead of replacing the existing one.

Reply:
Gotcha; the AC originally specified that `national` be replaced with `shortcode` but I can add it back